### PR TITLE
Group work v2 - PR for review comments

### PR DESCRIPTION
**Pull request by antoviaque**
Creating this fake PR to log comments on the group work v2 patch as a whole, to avoid polluting individual commits when the changes aren't in a currently open PR.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    - "2.7"
+before_install:
+    - "export DISPLAY=:99"
+    - "sh -e /etc/init.d/xvfb start"
+install:
+    - "pip install -e git://github.com/edx/xblock-sdk.git#egg=xblock-sdk"
+    - "pip install -r requirements.txt"
+    - "pip install -r test-requirements.txt"
+    - "pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements.txt"
+    - "pip install -r $VIRTUAL_ENV/src/xblock-sdk/test-requirements.txt"
```
Comment (antoviaque) [inline .travis.yml]: @e-kolpakov You might want to add a line similar to https://github.com/open-craft/problem-builder/blob/48fd595279493988393ef7dbbad4c228634dc304/.travis.yml#L12 to ensure this is installed the same was as in production -- currently it is installed with the `pip -e` option, which uses the development mode, and doesn't catch missing assets.

When installed with `python setup.py sdist && pip install dist/xblock-group-project-v2-0.2.tar.gz` it currently gives https://gist.github.com/antoviaque/1f7b051da6986ed9a256

Comment (e-kolpakov) [inline .travis.yml]: @antoviaque just rebuilt my env using this commands - it worked and didn't throw an exception you faced. I'm committing it to the branch under review to see if travis will complain about it.

Comment (antoviaque) [inline .travis.yml]: @e-kolpakov Try uninstalling the XBlock first (or not installing it at all in requirements.txt):

```
pip uninstall -y xblock-group-project-v2 && python setup.py sdist && pip install dist/xblock-group-project-v2-0.2.tar.gz
```

**Comment by e-kolpakov** [comment]
@antoviaque I believe we don't need this PR anymore - closing.

**Pull request closed without merging**